app/presenters/hyrax/member_presenter_factory.rb

```diff
@@ -68,5 +54,10 @@ def presenter_factory_arguments
       def composite_presenter_class
         CompositePresenterFactory.new(file_presenter_class, work_presenter_class, ordered_ids & file_set_ids)
       end
+
+      def file_sets
+        @file_set_ids ||= Hyrax::Queries.metadata_adapter_query_service.find_members(resource: @work,
```

Comment:
Should the instance variable be `@file_sets` since it is the objects and not the ids?